NeuroID/NeuroIDTracker.swift

```diff
@@ -147,7 +166,11 @@ public class NeuroIDTracker: NSObject {
             nidEvent.attrs = [attrVal, shVal]
 
             NeuroID.saveEventToLocalDataStore(nidEvent)
+
+            NIDPrintLog("*****************   Actually Registered View: \(dp.className) - \(dp.id)-")
```

Comment:
Are these debugging logs intended to be left in ?

Reply:
Yes, because they are NID controlled can be disabled by a configured setting. We often use this for helping int he debug process